code/Sitecore.NSubstitute/FakeItem.cs

```diff
@@ -34,6 +34,8 @@ public FakeItem(ID id = null, Database database = null)
       Item.Version.Returns(Version.First);
 
       this.Item.Children.Returns(new ChildList(this.Item, this.childList));

```

Comment:
Do we want to have a single childList object set for Children & GetChildren() & GetChildren(...) ?

In other words => obj.RefEquals(item.Children == item.GetChildren())?

Reply:
In Sitecore API we get new ChildList object every call. 
I do not see the reason for returning the same object all the time. Do you see any scenario where it would be useful?